index.html

```diff
@@ -4,10 +4,123 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Portfolio</title>
+  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Poppins" />
+  <!-- including css -->
+  <link rel="stylesheet" href="./CSS/index.css">
+  <link rel="stylesheet" href="./CSS/navbar.css">
+  <link rel="stylesheet" href="./CSS/images.css">
+  <link rel="stylesheet" href="./CSS/aboutme.css">
+  <link rel="stylesheet" href="./CSS/connections.css">
 </head>
 
 <body>
+ 
+    <nav class="navbar">
+      <div class="logo">MyLogo</div>
+
+      <div class="menu-burger-icon">
+        <div class="bar"></div>
+        <div class="bar"></div>
+        <div class="bar"></div>
+      </div>
```

Comment:
The hamburger button should be wrapped in a `nav` tag for semantic reasons. Kindly change the `div` tag to a `nav` tag.